difflint/lib.py

```diff
@@ -13,6 +14,29 @@
 MISSING_FILE_EXIT_CODE = 72  # os.EX_OSFILE is not portable
 
 
+def make_path_from_cwd(*args):
+    """
+    Builds a string containing the current working directory as the root
```

Comment:
nit: IMO, the current working directory isn't really the root of the path returned here. In your example, I would argue the root is actually "/". Consider saying something like: "Builds a filepath relative to the current working directory from zero or more given strings."